UI/user.html

```diff
@@ -241,16 +242,25 @@ <h1>EPIC Mail</h1>
         </div>
     </div>
     <script src="js/main.js"></script>
-
     <script>
-    function rowHide(){
-        var x = document.getElementById("rowHidden");
-        if (x.style.display === "none"){
-            x.style.display = "block";
-        } else {
-            x.style.display = "none";
+        function rowHide() {
```

Comment:
Is it possible to move this `js`  to a js file?
By this, I mean all the js functions here

Reply:
yes it is!